### DISCUSSION TITLE
question/wx/q1

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#967] [Announcements]
# question/wx/q1

提示 此组件包含头像裁剪，上传，查看大图等功能，支持微信小程序，H5，App，uni-app，nvue等平台 准备工作 微信公众号或者服务号 微信开放平台账号 前端工作 --- JSSDK使用步骤，见微信官方链接 --- https://developers.weixin.qq.com/doc/offiaccount/OA_Web_Apps/JS-SD...

http://localhost:8080/question/wx/q1.html

**Comment by nn11636924290** [comment]
321
